backend/mr-db-ebean/src/test/java/io/featurehub/db/GenerateDbMigration.java

```diff
@@ -13,6 +13,7 @@ public static void main(String[] args) throws Exception {
 
     dbMigration.setMigrationPath("/dbmigration");
     dbMigration.setStrictMode(false);
+//    dbMigration.setGeneratePendingDrop("1.3");
```

Comment:
do we need this commented out code here?

Reply:
It was just to remind me how it was done so when I have a pending drop I know how to force it to happen.